book/website/reproducible-research/licensing.md

```diff
@@ -10,102 +13,301 @@ No previous knowledge is needed, this chapter explains how important it is to un
+To attempt to achieve this goal, copy-left software requires that when distributing copy-left software or in some cases derived works that you do so under the same terms as the original licence.
+Creative commons 'share-alike' licences attempt the same thing for other content.
+
+Copy-left licences do not prohibit commercial use, indeed numerous companies exist which develop copy-left projects and sell support services for deploying those projects instead of using proprietary licences which would incentivise an unhealthy relationship with their users, [Nextcloud](https://nextcloud.com/) is an excellent example.
+Nextcloud makes use of the [AGPL v3.0](https://www.fsf.org/bulletin/2021/fall/the-fundamentals-of-the-agplv3) a license which extends the protections of the GPL to software used over a network.
+It gives users who interact with this software over a network, for example by using a web service, rather than run it on their own computers, the right to access a copy of the source code; which they are further free to modify and distribute as is usual for the GPL.
+
+Within copy-left licenses, there are 'strong' and 'weak' copy-left licences.
+'Strong' copy-left licenses require that combined works which contain them as a library also carry the same license but weak copy-left licenses permit their re-distribution as a library within a combined work under a different license.
+We will define derived and combined works in the section on licence compatibility where the detailed implications of the distinctions between strong & weak copyleft, and permissive licencing are explored in more practical detail.
+It is important to note that licenses can be incompatible such that creating a combined work is highly impractical to do legally.
+
+<table>
+    <thead>
+        <tr>
+            <th colspan="3">Free</th>
+            <th rowspan="3">Proprietary</th>
+        </tr>
+        <tr>
+            <th colspan="2">Copyleft</th>
+            <th rowspan="2">Permissive</th>
+        </tr>
+        <tr>
+            <th>Strong</th>
+            <th>Weak</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+        <td>GPL<sup>1</sup> CDDL<sup>2</sup></td>
+        <td>LGPL<sup>3</sup> MPL<sup>4</sup></td>
+        <td>BSD<sup>5</sup> MIT<sup>6</sup> Apache</td>
+            <td>Research Only: No&nbsp;copying, No&nbsp;modification</td>
+        </tr>
+    </tbody>
+    <caption>
+      <div class="footnote">
+        <sup>1</sup>GPL: GNU General Public License <sup>2</sup>CDDL: Common Development and Distribution License <sup>3</sup>LGPL: GNU Lesser General Public License <sup>4</sup> MPL: Mozilla Public License <sup>5</sup> BSD: Berkeley Software Distribution <sup>6</sup> MIT: Massachusetts Institute of Technology
+      </div>
+      Licenses can either be Free or Proprietary, with Free Licenses further classified as Copyleft or Permissive.
+    </caption>
+</table>
```

Comment:
Is the html here for the footnotes?

Maybe the same thing can be done with just the MyST syntax

- https://mystmd.org/guide/typography#footnotes
- https://mystmd.org/guide/tables#adding-a-caption

If not, it might be worth thinking of another way to present the same information.

Reply:
I copy pasted this from the previous version of this section, I can try my hand at an MyST native table if that is preferable